test/e2e/setup.sh

```diff
@@ -185,7 +185,8 @@ do
             --set enableV1Alpha1APIs=false \
             --set enableV1Beta1APIs=true \
             --set propertyProvider=$PROPERTY_PROVIDER \
-            --set region=${REGIONS[$i]}
+            --set region=${REGIONS[$i]} \
+            -f azure_valid_config.yaml
```

Comment:
it depends on the property_provider. only when property_provider is azure, we need to use azure_valid_config.yaml